src/pages/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import StocksList from '../components/stocks/StocksList';
+
+const Home = React.memo(() => (
+  <section>
+    <StocksList />
```

Comment:
- [ ] Instead of displaying one of the stocks as the intro section, kindly refactor it to display a more descriptive message showing what the items on the page are intended for, just like the design. This will enhance the user experience.

  ![image](https://github.com/lucy-sees/Stocks-API-App/assets/54848363/05537564-8746-4076-87e2-23d8126ac550)